apps/drec-api/migrations/devices.json

```diff
@@ -0,0 +1,86 @@
+[
+  {
+    "id": 2,
+    "drecID": "DREC02",
+    "registrant_organisation_code": "D0012",
+    "project_name": "Project Device 1",
+    "address": "Street 1, Bucharest",
+    "latitude": "34.921213",
+    "longitude": "135.717309",
+    "fuel_code": "ES200",
+    "device_type_code": "T020001",
+    "installation_configuration": "StandAlone",
+    "capacity": 1500,
+    "commissioning_date": "2012-07-01",
+    "grid_interconnection": false,
+    "off_taker": "School",
+    "sector": "Education",
+    "standard_compliance": "I-REC",
+    "yield": 1000,
```

Comment:
I would consider twice using `yield` name for that field, because it's keyword in javascript and it may complicate some things. Up to you :)

Reply:
good point